src/app/authoring-tool/edit-component-advanced/edit-component-advanced.component.html

```diff
@@ -0,0 +1,107 @@
+<h1 mat-dialog-title i18n>Advanced Settings</h1>
+<div mat-dialog-content>
+  <div class="md-dialog-content">
+    <ng-container [ngSwitch]="component.content.type">
+      <edit-animation-advanced
+        *ngSwitchCase="'Animation'"
+        [nodeId]="nodeId"
```

Comment:
Get nodeId from component for all the components?
```suggestion
        [nodeId]="component.nodeId"
```